packages/lmd/kibana/security_rule/2577e749-ed63-4b58-ac32-4f8a94d73e60.json

```
@@ -16,7 +16,8 @@
         "tags": [
             "Elastic",
             "Lateral Movement Detection",
-            "host"
+            "host",
```

Let's keep all tags in title case for consistency.
```suggestion
            "Host",
```